src/extension.ts

```diff
@@ -1506,7 +1513,9 @@ async function setup(context: vscode.ExtensionContext, progress?: ProgressHandle
 
   // List of functions that will be bound commands
   const funs: (keyof ExtensionManager)[] = [
-    "useCMakePresets",
+    'activeFolderName',
+    'activeFolderPath',
+	  "useCMakePresets",
```

Comment:
Indentation?

Reply:
Interesting, in Vscode (1.54.1) this is not visible.